### PR TITLE
Area booking and controlling policy updates

### LINKED ISSUE #1747: Primary and secondary sectors definition
**Div Pol**

3.2 b For the purpose of this and other documents, the VATSIM UK website shall define ’primary
sectors’ and ‘secondary sectors’ for the EGTT and EGPX FIR.

**Primary Sectors**

LON_S_CTR
LON_C_CTR
LON_W_CTR
LON_N_CTR
SCO_CTR

**Secondary Sectors**

LON_D_CTR
LON_E_CTR
LTC_S_CTR
LTC_N_CTR
MAN_CTR
STC_CTR

### PR DESCRIPTION
**Pull request by hsugden**
Resolves #1747

### CODE REVIEW

**Comment by hsugden** [comment]
@cpawley could you perhaps check this is correct, please?

File: resources/views/site/operations/sectors.blade.php [hunk 1]
```diff
@@ -316,12 +316,13 @@
...
-                    <p>
-                        <strong>Primary Sectors</strong>
-                    </p>
+                    <h2>
+                        Information for Controllers
+                    </h2>
 
                     <p>
-                        For the purpose of controller bookings, the &#39;Primary Sectors&#39; are defined as:
+                        For the purpose of <a "{{ route('site.atc.bookings') }}" rel="">controller bookings</a>,
```
Comment (atoff) [inline resources/views/site/operations/sectors.blade.php:324]:
```suggestion
                        For the purpose of <a href="{{ route('site.atc.bookings') }}">controller bookings</a>, 
```

Comment (hsugden) [reply resources/views/site/operations/sectors.blade.php:324]: Good with me! 

href error is my fault. 

rel="" was taken from elsewhere though! https://github.com/VATSIM-UK/core/blob/master/resources/views/site/atc/progression.blade.php#L22

Comment (atoff) [reply resources/views/site/operations/sectors.blade.php]: Yeah, it doesn't really make any difference, and in this context it doesn't do anything. Not sure why it is above!

**Pull request merged**